doggos.md

```diff
@@ -1 +1,2 @@
 henry of henrythecoloradodog
+maya of mayapolarbear
```

Comment:
maya is the cutest

Reply:
she trained in puppy school for 2 months

Reply:
omg that so much training 🙀